pages/c/_cluster/apps/charts/index.vue

```diff
@@ -590,4 +689,108 @@ export default {
   }
 }
 
+.slider {
```

Comment:
Please remove style duplication and component specific styling.

Reply:
Fixed